src/bindings/jna/libelektra/src/main/java/org/libelektra/Key.java

```diff
@@ -328,6 +328,19 @@ public Key setBinary(byte[] value) {
     checkReturnValue(Elektra.INSTANCE.keySetBinary(getPointer(), value, value.length));
     return this;
   }
+  
+  /**
+   * Removes the key's value without changing the type
+   *
+   * @return This {@link Key}, enabling a fluent interface
+   * @throws IllegalStateException if this {@link Key} has already been released
+   * @throws KeyException if the key's value is read-only or there have been allocation problems
+   */
+  @Nonnull
+  public Key setNull() {
+	checkReturnValue(isBinary() ? Elektra.INSTANCE.keySetBinary(getPointer(), null, 0) : Elektra.INSTANCE.keySetString(getPointer(), null) );
```

Comment:
```suggestion
	checkReturnValue(Elektra.INSTANCE.keySetBinary(getPointer(), null, 0));
```

The function needs to turn the key into a binary one, because the current implementations of `keyString`, `keyGetValueSize`, etc. all behave as if the value was `""`, when a string key is set to `NULL`.